pytorch_transformers/tokenization_utils.py

```diff
@@ -666,35 +696,146 @@ def _convert_token_to_id(self, token):
     def encode(self, text, text_pair=None, add_special_tokens=False, **kwargs):
```

Comment:
Didn't we say that `encode` should now just be a front end to `encode_plus` filtering the output?

Reply:
You're right, it's cleaner this way